src/main/java/com/test/fibonacci/businesslogic/entity/Response.java

```diff
@@ -0,0 +1,29 @@
+package com.test.fibonacci.businesslogic.entity;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import java.util.List;
+
+/**
+ * Holds the fields that can be returned when making a request to any of the endpoints in this application.
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder
+public class Response {
+
+    @Setter
+    @Getter
+    private Long nextNumber;
+    @Setter
+    @Getter
+    private List<Long> fibonacciSequence;
+    @Setter
+    @Getter
+    private String message;
+}
```

Comment:
Folosind adnotarea `@Data` poti evita necesitatea de a folosi `@NoArgsConstructor`, `@AllArgsConstructor` si `@Getter`/`@Setter`. Poti merge un pas si mai departe tinand cont ca folosesti `@Builder` si sa faci structura de date immutable folosind `@Value` in loc de `@Data`.

In alta ordine de idei, nu prea sunt fan al idei de a amesteca campurile specifice raspunsului pentru  mai multe endpoint-uri intr-un singur `DTO` (single responsability and all that stuff 😁 )